asttokens/util.py

```diff
@@ -36,6 +37,11 @@ class EnhancedAST(AST):
 
   AstNode = Union[EnhancedAST, NodeNG]
 
+  if sys.version_info[0] == 2:
```

Comment:
Please add `  # pragma: no cover` to `if TYPE_CHECKING:` above to solve the reduction in coverage in this PR.

Reply:
4e931c6bb2b9ef683761a544c1645a8d12fd9b67